docs/adding_a_dataset.md

```diff
@@ -73,15 +71,15 @@ model = mteb.get_model("intfloat/multilingual-e5-small")
 results = mteb.evaluate(model, tasks=[SciDocsReranking()])
 ```
 
-> **Note:** for multilingual / crosslingual tasks, make sure you've specified `eval_langs` as a dictionary, as shown in [this example](../mteb/tasks/classification/multilingual/MTOPIntentClassification.py).
+> **Note:** for multilingual / crosslingual tasks, make sure you've specified `eval_langs` as a dictionary, as shown in [this example](../mteb/tasks/classification/multilingual/mtop_intent_classification.py).
 
 
 
 ### A Detailed Example
 Often the dataset from HuggingFace is not in the format expected by MTEB. To resolve this you can either change the format on Hugging Face or add a `dataset_transform` method to your dataset to transform it into the right format on the fly. Here is an example along with some design considerations:
 
 ```python
-class VGClustering(AbsTaskClusteringLegacy):
+class VGClustering(AbsTaskClustering):
```

Comment:
Should we simplify the example a bit?

```
class VGClustering(AbsTaskClustering):
   metadata = ...
   
   def load_data():
      # hacky first version that just load labels and texts
```

which we can then follow up with a `push_dataset_to_hub` which allows us to remove the custom load_data and format it correctly.

We could even do this for a couple of task types (selectable using tab, similar to cli/python)